Datapack/data/dvz/functions/zombies/enchanter/enchant_loop.mcfunction

```diff
@@ -0,0 +1,25 @@
+# -------------------------------------------
+# Called By: dvz:zombies/enchanter/enchant_loop
+# File Name: enchant_loop
+# File Purpose: Enchanted play loop for enchanter
+# Created By: Galactic
+# 
+# Created On: 2023.06.26
+# Last Modified On: 2023.06.26
+# Last Modified By: Galactic
+#
+# Credit to: Galactic
+#
+# Comments: 
+# -------------------------------------------
+
+execute as @s[scores={DVZ.enchanterloop.ticks=1..}] at @s run particle minecraft:large_smoke ~ ~0.9 ~ 0.5 1 0.5 0 1
+execute as @s[scores={DVZ.enchanterloop.ticks=1..}] at @s run particle minecraft:dust 0.5 0.1 0.1 1.5 ~ ~0.9 ~ 0.5 1 0.5 0 1
+execute as @s[scores={DVZ.enchanterloop.ticks=1..}] at @s run effect give @s glowing 1 0
+execute as @s[scores={DVZ.enchanterloop.ticks=1..}] at @s run effect give @s haste 1 2
+execute as @s[scores={DVZ.enchanterloop.ticks=1..}] at @s run effect give @s resistance 1 1
+execute as @s[scores={DVZ.enchanterloop.ticks=1..}] at @s run attribute @s minecraft:generic.attack_damage modifier add 23df59fa-29af-4c00-91dd-dad35afbb73d enchanter_damage 5 add
```

Comment:
this is run every tick for 25 ticks. Perhaps a better way, so it's only applied once?

Reply:
You could have it set at tick 25, with `enchant_pulse` and this file is only used to remove?

Reply:
Seems safer to put it on a loop, might cause unwanted behaviour otherwise